### PR TITLE
feat: add page to check the PR results in this repo

### PR DESCRIPTION
**Pull request by galanggg**
Hi everyone, this is my PR to calculate how many PR has been done in this repo. Refer this issue #16 
Preview [Link](https://raw.githack.com/galanggg/open-source/76325c5a890d89629f85f939223a52bba1d66067/en_ID/pages/pr-check.html)

### CODE REVIEW

File: en_ID/pages/internship.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following are the required hardware components that will be needed as part o
...
+Because these steps are simple, we expect high-quality work, which may take a longer time. We want to see that you are capable of using or learning how to use these tools (writing good GitHub issues, creating pull requests, using Markdown, Docker, etc). These steps may seem easy, but we want you to impress us with good GitHub etiquette and quality Markdown. The bare minimum would be just to passively follow the steps; you should do further reading about tools/language we use so you can further your understanding and relieve confusion if you're unclear about how something works. **Treat these steps as learning opportunities!**. The GitHub and Markdown skills you practice here are very important for both this internship and a future software development career. [Check the progress PR](pr-check.html)
```
Comment (empeje) [inline en_ID/pages/internship.md]: I think you can add the link in the navigation menu as well.

**Comment by galanggg** [comment]
but i have some problem with coma when results comes out, like this
localhost_5000_en_ID_pages_pr-check

File: en_ID/pages/assets/script.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  }
+}
+
+const showData = () => {
+  checkPullRequests(username.value).then((res) => {
+    console.log(res)
+    const loop = res.items.map((user) => {
+      return `<li>${user.title}</li>`
+    })
+    resultsContainer.innerHTML = loop
```
Comment (empeje) [inline en_ID/pages/assets/script.js]: So the loop is an array of string, if you do the `resultsContainer.innerHTML = loop` directly the comma that coming from the array is inherited.

**Review by empeje — CHANGES_REQUESTED** [review]

**Comment by galanggg** [comment]
here the update [preview](https://raw.githack.com/galanggg/open-source/ec345d6d5575049c567718471e002fb2894ab2e4/en_ID/#!pages/progress.md), please to checks and reviews

**Comment by empeje** [comment]
Tanks @galanggg, your PR is amazing, the next step is how to display a summary

e.g.

PR 4/5
Issues 3/4
Comments 4/4

**Comment by galanggg** [comment]
> Tanks @galanggg, your PR is amazing, the next step is how to display a summary
> 
> e.g.
> 
> PR 4/5
> Issues 3/4
> Comments 4/4

Thanks for the feedback @empeje , i will try for the summary !

**Comment by empeje** [comment]
Hi @galanggg any update on the summary?

Something simple like the following is enough

```
Progress for empeje:
Number of Comments: 0

Number of Issues: 0

Number of Merged PRs: 0

```

**Comment by galanggg** [comment]
> Hi @galanggg any update on the summary?
> 
> Something simple like the following is enough
> 
> ```
> Progress for empeje:
> Number of Comments: 0
> 
> Number of Issues: 0
> 
> Number of Merged PRs: 0
> ```

hi @empeje , im still working on it :smile:

**Review by empeje — APPROVED** [review]
Looks good 🎉

**Pull request merged**